### ISSUE TITLE
rsync: link_stat "/Users/sreeram/Downloads/ez_flutter-Sagar/build/ios/Debug-iphonesimulator/App.framework" failed: No such file or directory (2)

### ISSUE THREAD

**Issue by PrasadPanyam** [flutter/flutter#83771]
Launching lib/main.dart on iPhone 12 Pro Max in debug mode...
Running Xcode build...
Xcode build done.                                           13.0s
Failed to build iOS app
Error output from Xcode build:
↳
    ** BUILD FAILED **

Xcode's output:
↳
    building file list ... rsync: link_stat "/Users/sreeram/Downloads/ez_flutter-Sagar/build/ios/Debug-iphonesimulator/App.framework" failed: No such file or directory (2)
    done

    sent 29 bytes  received 20 bytes  98.00 bytes/sec
    total size is 0  speedup is 0.00
    rsync error: some files could not be transferred (code 23) at /System/Volumes/Data/SWE/macOS/BuildRoots/2288acc43c/Library/Caches/com.apple.xbs/Sources/rsync/rsync-55/rsync/main.c(996) [sender=2.6.9]
    Command PhaseScriptExecution failed with a nonzero exit code
    note: Using new build system
    note: Building targets in parallel
    note: Planning build
    note: Constructing build description
    warning: The iOS Simulator deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 14.4.99. (in target 'FMDB' from project 'Pods')
    warning: The use of Swift 3 @objc inference in Swift 4 mode is deprecated. Please address deprecated @objc inference warnings, test your code with “Use of deprecated Swift 3 @objc inference” logging enabled, and then disable inference by changing the "Swift 3 @objc Inference" build setting to "Default" for the "Runner" target. (in target 'Runner' from project 'Runner')
    warning: The iOS Simulator deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 14.4.99. (in target 'Flutter' from project 'Pods')

Could not build the application for the simulator.
Error launching application on iPhone 12 Pro Max.

**Comment by PrasadPanyam** [comment]
doctor --verbose
[✓] Flutter (Channel stable, 2.2.1, on macOS 11.3.1 20E241 darwin-x64, locale en-US)
    • Flutter version 2.2.1 at /Users/sreeram/Desktop/Application/flutter
    • Framework revision 02c026b03c (5 days ago), 2021-05-27 12:24:44 -0700
    • Engine revision 0fdb562ac8
    • Dart version 2.13.1

[✓] Android toolchain - develop for Android devices (Android SDK version 30.0.3)
    • Android SDK at /Users/sreeram/Library/Android/sdk
    • Platform android-30, build-tools 30.0.3
    • Java binary at: /Applications/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6915495)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 12.4, Build version 12D4e
    • CocoaPods version 1.10.1

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[✓] Android Studio (version 4.1)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6915495)

[✓] VS Code (version 1.55.2)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.22.0

[✓] Connected device (2 available)
    • iPhone 12 Pro Max (mobile) • E9EC1A99-8A10-4328-8185-E88E6B258D28 • ios            • com.apple.CoreSimulator.SimRuntime.iOS-14-4 (simulator)
    • Chrome (web)               • chrome                               • web-javascript • Google Chrome 91.0.4472.77
    ! Error: Sreeramulu's iPhone is not connected. Xcode will continue when Sreeramulu's iPhone is connected. (code -13)

• No issues found!
Process finished with exit code 0

**Comment by darshankawar** [comment]
@PrasadPanyam 
Can you provide complete `flutter run --verbose` log ? It will be large, so you can put that in a file and attach it here.

Also, provide `ios/Runner.xcodeproj/project.pbxproj`.

**Comment by PrasadPanyam** [comment]
flutter run --verbose
[flutter run –verbose.docx](https://github.com/flutter/flutter/files/6584719/flutter.run.verbose.docx)

**Comment by darshankawar** [comment]
@PrasadPanyam 
The log you provided may not reflect the error you mentioned in original comment. In the above log, it failed because of below:

```
[ +138 ms] No supported devices connected.
[ +105 ms] The following devices were found, but are not supported by this project:
                    Chrome (web) • chrome • web-javascript • Google Chrome 91.0.4472.77
                    If you would like your app to run on web, consider running `flutter create .` to generate projects for these platforms.

```
Is this your current error due to which you aren't able to run ? 
Else, attach a physical device or simulator and run the verbose command that'll show your original error you mentioned.

**Comment by PrasadPanyam** [comment]
@darshankawar 
Thanks for your response, but I didn't run this application on the web. I am using an ios simulator.

**Comment by darshankawar** [comment]
Yes, but the verbose log doesn't show `iphonesimulator/App.framework" failed: No such file or directory (2)` entry which you posted earlier.

**Comment by darshankawar** [comment]
Without additional information, we are unfortunately not sure how to resolve this issue. 
We are therefore reluctantly going to close this bug for now. 
Please don't hesitate to comment on the bug if you have any more information for us; we will reopen it right away!  
Thanks for your contribution.

Could everyone who still has this problem please file a new issue with the exact description of what happens, logs, and the output of `flutter doctor -v`.
All system setups can be slightly different, so it's always better to open new issues and reference related issues.
